handlers.go

```diff
@@ -26,11 +26,13 @@ func (sf *Sunfish) AddFile(w http.ResponseWriter, r *http.Request) {
 	body, err := ioutil.ReadAll(io.LimitReader(r.Body, maxSiaFilesize))
 
 	if err != nil {
-		panic(err)
+		sf.logger.Println("ERROR: Could not ready body of request.")
```

Comment:
If the file is too big, AKA it's not a real Siafile, not sure what an empty request would do. But why does that manner?